src/components/CDPList.js

```diff
@@ -41,17 +84,52 @@ const NavbarItem = ({ href, label, ratio, owned, active, ...props }) => (
   </NavbarItemContainer>
 );
 
+const CdpContainer = styled(Flex)`
+  cursor: pointer;
+  flex-direction: column;
+  overflow: auto;
+  height: ${props => (props.cdpsLength >= 4 ? '250px' : undefined)};
+`;
+
+// replace Fragment with this & add style={props}
+// const AnimatedWrap = styled(animated.div)`
+//   width: 100%;
+// `;
+
 const CDPList = memo(function({ currentPath, viewedAddress, currentQuery }) {
+  const { url } = useCurrentRoute();
+  const [listOpen, setListOpen] = useState(false);
   const { maker, account } = useMaker();
   const [{ cdps, feeds }, dispatch] = useStore();
   const [ratios, setRatios] = useState([]);
   const [navbarCdps, setNavbarCdps] = useState([]);
+  const [overviewPath, setOverviewPath] = useState(currentPath);
+  const active = currentPath === overviewPath;
+
+  // const fadeProps = {
+  //   to: { opacity: listOpen ? 1 : 0 },
+  //   from: { opacity: listOpen ? 0 : 1 }
+  // };
+  // const [props, set, stop] = useSpring(
+  //   () => console.log('animation render2', listOpen) || fadeProps
+  // );
+
+  useEffect(() => {
+    const onSavePage = url.pathname === `/${Routes.SAVE}`;
```

Comment:
we might want a `useMemo` here since `listOpen` is function of the props rather than a side effect